config/packages/security.yaml

```
@@ -22,6 +22,7 @@ security:
     access_control:
         - { path: ^/admin, roles: ROLE_ADMIN }
         - { path: '^/regulations/([0-9a-f]{8}-[0-9a-f]{4}-[13-8][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12})$', methods: [GET], roles: PUBLIC_ACCESS }
+        - { path: '^/regulations', methods: [GET], roles: PUBLIC_ACCESS }
```

Ici il faudrait uniquement autoriser l'accès public à cette page. En l'état, tout ce qui se trouve derrière (par exemple `/regulations/{uuid}/duplicate` sera ouvert).

```suggestion
        - { path: '^/regulations$, methods: [GET], roles: PUBLIC_ACCESS }
```

---

Et j'imagine que c'est bien compatible avec la ligne suivante ? : 
```yaml
        - { path: '^/(_fragment|regulations|feedback)', roles: ROLE_USER }
```

---

Oui, pour tout ce qui ne match pas ta première regex, il passera ici